localization/english/event_localization/wc_draenor_events_l_english.yml

```diff
@@ -40,26 +40,26 @@
+ WCDRA_1000_DESC_rift:0 "The day that all denizens of Azeroth feared has come - the Dark Portal has been opened once more! The Elder Shaman Ner’zhul and his followers on Draenor have stabilized the rift from the other side and sent his forces into the Blasted Lands, right under [portal_owner.GetTitledFirstNamePossessive] nose!\n\n"
  WCDRA_1000_DESC_ending:0 "Through trickery and guile, Ner’zhul’s most trusted followers slipped past the defenders of the [portal_owner.Custom('GetPortalRift')] and set forth to recover a collection of powerful artifacts: the Book of Medivh, the Jeweled Scepter of Sargeras, and the Eye of Dalaran. No one can say for sure what he plans to do with all these artifacts, but all who call this world their home agree that he must be stopped at all costs!"
  WCDRA_1000_OPT_A:0 "I shall call upon the [ROOT.Char.Custom('KnightCulturePluralNoTooltip')|l] of the realm!"
  WCDRA_1000_OPT_B:0 "I will lead the charge myself!"
- WCDRA_1000_OPT_C:0 "This does not concern us…"
+ WCDRA_1000_OPT_C:0 "This does not concern us..."
  WCDRA_1000_OPT_C_unable:0 "Alas, we can do nothing."
  WCDRA_1000_OPT_C_frostwolf:0 "What is Ner’zhul planning now?"
  WCDRA_1000_OPT_C_orcish:0 "Lok’tar Shadowmoon!"
  
  WCDRA_1005_DESC_begging:0 "Our warriors have successfully stormed through the Dark Portal and taken the Horde fortress known as Hellfire Citadel. The battle was hard-fought and bloody, but ultimately we were able to seize victory.\n\n"
  WCDRA_1005_DESC_injured:0 "Among others, [draenor_wounded.GetFullNameNicknamed] was left gravely wounded. Were it not for the skill of our healers, [draenor_wounded.GetSheHe] may very well have lost [draenor_wounded.GetHerHis] life in this battle."
- WCDRA_1005_DESC_killed:0 "Not all were so fortunate, in the end. [draenor_killed.GetFullNameNicknamed] paid the ultimate price for this victory, and they will be remembered with honor. Alas, there is no time to mourn, for we must press on if their sacrifice is to have any meaning."
+ WCDRA_1005_DESC_killed:0 "Not all were so fortunate, in the end. [draenor_killed.GetFullNameNicknamed] paid the ultimate price for this victory and will be remembered with honor. Alas, there is no time to mourn, for we must press on if their sacrifice is to have any meaning."
  empty_line:0 "\n\n"
  WCDRA_1005_DESC_ending:0 "Before he was slain by [wise_mage.GetFullNameNicknamed], the ogre magi Dentarg revealed that the Elder Shaman seeks to open new portals to other worlds for the Horde to invade!\n\n[wise_mage.GetFirstName] believes that the Dark Portal could be sealed forever if we could procure the Book of Medivh from Ner’zhul himself, as well as the Skull, which is currently held by the Black Dragon Aspect, Deathwing. The Destroyer seems to be in league with the Horde - we will have to defeat him to claim the Skull!"
  THEY_GOT_ME:0 "#emp They got me...#!"
  WCDRA_1005_OPT_A_dragon:0 "Neltharion’s treachery knows no bounds!"
```

Comment:
Shouldn't we use either Neltharion OR Deathwing in one event, instead of both names? (Suggested by panpoleon)

Reply:
> Shouldn't we use either Neltharion OR Deathwing in one event, instead of both names? (Suggested by panpoleon)

It's an option for dragons. They know his real name and thus refer to him as Neltharion.